Makefile

```diff
@@ -105,6 +105,14 @@ build-release-tarballs: ## Create a series of `.tar.gz` files in the BIN_DIR dir
 
 ##@ Test
 
+.PHONY: unit
+unit: ## Run unit tests with coverage.
+	cargo llvm-cov nextest --html --locked --workspace --all-features -E 'kind(lib)' -E 'kind(bin)' -E 'kind(proc-macro)'
+
+.PHONY: html
+html: ## Open coverage report in browser.
+	open target/llvm-cov/html/index.html
```

Comment:
this depends on unit, right?

Reply:
Yes exactly. I wasn't sure whether you guys preferred to run the tests and open the coverage report in one command or two.